scripts/deploy-helm.sh

```diff
@@ -234,7 +234,7 @@ if [[ $DEPLOYMENT_TYPE == all ]] || [[ $DEPLOYMENT_TYPE == logs ]]; then
   if [[ $USE_EXISTING_ACTIVE_GATE == false ]]; then
     check_dynatrace_docker_login
   else
-    echo "Using an existing Active Gate"
+    echo "Using an existing Active Gate" | tee -a "$FULL_LOG_FILE"
     check_if_parameter_is_empty "$DYNATRACE_LOG_INGEST_URL" "DYNATRACE_LOG_INGEST_URL"
     check_url "$DYNATRACE_LOG_INGEST_URL" "$DYNATRACE_URL_REGEX" "$ACTIVE_GATE_TARGET_URL_REGEX" \
```

Comment:
I'm looking at examples where `tee` is not used, as you advised. Here we're not using it neither at call site nor in the `check_url` implementation. What's the reason?

Reply:
because in `check_url` function we are using `err` function to return message and there `tee` is added

Reply:
this is better example when we want to have information in file but on in stdout https://github.com/dynatrace-oss/dynatrace-gcp-function/pull/267/files#diff-5838719667e50a69f27dfddac131cad8ed4e11bee597a64a1b48454efdfad40bR217